mysql/check.py

```diff
@@ -1023,6 +1023,9 @@ def _get_stats_from_innodb_status(self, db):
                 elif len(row) == 18:
                     results['Innodb_pending_normal_aio_reads'] = long(row[4])
                     results['Innodb_pending_normal_aio_writes'] = long(row[12])
+                elif len(row) == 22:
```

Comment:
could you please add the format in the commented section above.... Man there are way too many formats 😅